airflow/operators/bash.py

```diff
@@ -95,10 +101,16 @@ class BashOperator(BaseOperator):
 
         bash_task = BashOperator(
             task_id="bash_task",
-            bash_command='echo "here is the message: \'$message\'"',
+            bash_command=['echo', "here is the message: \'$message\'"],
             env={'message': '{{ dag_run.conf["message"] if dag_run else "" }}'},
         )
 
+    We also highly recommend against using [\"bash\", \"-c\"... in your command as that command will negate
```

Comment:
Don't need `\` here

Reply:
Perhaps we can link to some docs here?  I am willing to write up a page on injection in general, and why parallelization is a good tool for the job.

Reply:
Doc PRs are always appreciated!